project/Base.scala

```diff
@@ -133,6 +133,26 @@ class Base extends Build {
        |""".stripMargin.split("\n").toSeq
 
   val nodeModulesRE = ".*/node_modules/.*".r
+
+  // This is a workaround for https://github.com/twitter/finagle/issues/688 and should be removed
+  // once that issue is fixed.
```

Comment:
would you mind briefly describing the problem/workaround.

```
/**
 * finagle-thrift-18.04 incorrectly exports symbols from libthrift-0.10.0.
 * In order to build assembly packages, we chose an arbitrary copy of these files
 * on the assumption that they are identical.
 *
 * This can be safely removed once twitter/finagle#688 is resolved.
 */
```

... or something